project_task_id_display/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+Project Task ID Display
+=======================
+
+This module extends the functionality of project to support add id to task kanban view.
```

Comment:
This module extends the functionality of project and displays the corresponding id to task kanban view.